pkgs/applications/window-managers/sway/default.nix

```diff
@@ -8,8 +8,16 @@
 , isNixOS ? false
 
 , enableXWayland ? true
+, systemdSupport ? stdenv.isLinux
+, dbusSupport ? true
+, trayEnabled ? dbusSupport
 }:
 
+# sway's meson_options.txt does not include a "none" option for
+# "sd-bus".  However if you choose "basu" and disable tray support, it
+# will build without needing basu (neither at build time nor run time).
+assert !systemdSupport && !dbusSupport -> !trayEnabled;
```

Comment:
Shouldn't this be the other way around? 

`trayEnabled -> systemdSupport && dbusSupport`

Currently as it written it would imply that _either_ systemd _or_ dbus is required for tray support, isn't it both?

Reply:
Wow, thanks.  I got the [contrapositive](https://en.wikipedia.org/wiki/Contraposition) wrong.  Corrected in latest push.

Reply:
(even if it's either-or I still think `trayEnabled -> systemdSupport || dbusSupport` would be clearer)